### ISSUE TITLE
Code for fine-tuning the model for sequence classification

### ISSUE THREAD

**Issue by rastadayon** [AI21Labs/sense-bert#11]
Hi. Thank you very much for sharing the code for loading your models. I am trying to use the SenseBert model to get the sense of some words in tweets and for that, I want to fine-tune your model for a simple sequence classification that classifies tweets into offensive and non-offensive. Could you share the code that would be able to do this or tell me how I could do this?

Thanks in advance!

**Comment by saptarshi059** [comment]
Yup, I also have the same question. I am trying to fine-tune the model on SQuAD but the readme doesn't seem to provide instructions on the same. Have you figured out how to do it?
